snippets/language-asciidoc.cson

```diff
@@ -44,10 +44,10 @@
       ----
       $0
       '''
-  'Code Block':
-    description: 'Code Block'
+  'Source Block':
+    description: 'Source block'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#listing-blocks'
-    prefix: '--s'
+    prefix: '``'
```

Comment:
Thinking more about it, `--s` does really fit because it's a modifier on another type of block, hence the extra char.